add geolocation samples (#944)
GoogleChrome/chrome-extensions-samples: functional-samples/cookbook.geolocation-popup/README.md

This recipe shows how to get `geolocation` access within a popup

## Context

> In extensions, geolocation uses the same web APIs you are used to. Instead, if you want to get geolocation information, use the same [`navigator.geolocation`](geolocation) DOM API that any web site normally would. This demo shows how to access this data in a [popup][3]. We also have demos for a [Service Worker][5](via [offscreenDocument][2], and using a [content script][4]

Reviewer: ```suggestion
To get geolocation information in extensions, use the same [`navigator.geolocation`][6] DOM API that any website normally would.This demo shows how to access this data in a [popup][3]. We also have demos for a [Service Worker][5](via [offscreenDocument][2], and using a [content script][4]
```

## Running this extension

1. Clone this repository.
1. Load this directory in Chrome as an [unpacked extension][2].
1. Open the Extension menu and click the extension named "Geolocation - popup".

[1]: https://developer.chrome.com/docs/extensions/mv3/getstarted/development-basics/#load-unpacked
[2]: https://developer.chrome.com/docs/extensions/reference/offscreen/
[3]: functional-samples/cookbook.geolocation-popup
[4]: functional-samples/cookbook.geolocation-contentscript

> [5]: functional-samples/cookbook.geolocation-offscreen

Reviewer: ```suggestion
[5]: functional-samples/cookbook.geolocation-offscreen
[6]: https://developer.mozilla.org/docs/Web/API/Navigator/geolocation
```

This recipe shows how to get `geolocation` access within a popup

## Context

> To get geolocation information in extensions, use the same [`navigator.geolocation`][6] DOM API that any website normally would.This demo shows how to access this data in a [popup][3]. We also have demos for a [Service Worker][5](via [offscreenDocument][2], and using a [content script][4]

Author: ditto to my question about what is being dittoed

> To get geolocation information in extensions, use the same [`navigator.geolocation`][6] DOM API that any website normally would.This demo shows how to access this data in a [popup][3]. We also have demos for a [Service Worker][5](via [offscreenDocument][2], and using a [content script][4]

Reviewer 2: Haha, still this one: https://github.com/GoogleChrome/chrome-extensions-samples/pull/944#discussion_r1231148022

> To get geolocation information in extensions, use the same [`navigator.geolocation`][6] DOM API that any website normally would.This demo shows how to access this data in a [popup][3]. We also have demos for a [Service Worker][5](via [offscreenDocument][2], and using a [content script][4]

Author: :D for the sake of redundancy...https://github.com/GoogleChrome/chrome-extensions-samples/pull/944#discussion_r1231245539

> This recipe shows how to get `geolocation` access within a popup

Reviewer 2: Nit: Can we move this down between the header and "Overview"?

Author: fixed with [1dbac70](https://github.com/GoogleChrome/chrome-extensions-samples/pull/944/commits/1dbac7034801f93237ddfa43f3a9a44f9f7552eb)

# geolocation inside a popup

## Overview

> To get geolocation information in extensions, use the same [`navigator.geolocation`][6] DOM API that any website normally would.This demo shows how to access this data in a [popup][3]. We also have demos for a [Service Worker][5](via [offscreen document][2], and using a [content script][4]

Reviewer 2: ```suggestion
To get geolocation information in extensions, use the same [`navigator.geolocation`][6] DOM API that any website normally would. This demo shows how to access this data in a [popup][3]. We also have demos for a [Service Worker][5](via [offscreen document][2]), and using a [content script][4]
```

Author: fixed with [1dbac70](https://github.com/GoogleChrome/chrome-extensions-samples/pull/944/commits/1dbac7034801f93237ddfa43f3a9a44f9f7552eb)
